include/RAJA/pattern/launch/launch_core.hpp

```diff
@@ -418,7 +476,7 @@ RAJA_HOST_DEVICE RAJA_INLINE void tile_icount(CONTEXT const &ctx,
                                                           body);
 }
 
-}  // namespace expt
+} //namespace expt
```

Comment:
Do these `loop` and `tile` policies need to be taken out of `::expt`?

Reply:
No, these were an initial prototype for perfectly nested loops